ClavaLaraApi/src-lara-clava/clava/clava/code/Inliner.js

```diff
@@ -0,0 +1,107 @@
+laraImport("clava.ClavaJoinPoints");
+
+class Inliner {
+  #variablePrefix;
+  #variableIndex;
+  constructor() {
+    this.#variableIndex = 0;
+    this.#variablePrefix = "__inline";
```

Comment:
If we successively inline several calls inside calls, this means that we can get progressively larger variables names, like __inline__inline__inline_a_0_1_2?

Should this be parameterizable, or it is important to control this name?